src/components/shared/Table.js

```diff
@@ -1,42 +1,67 @@
 import React from "react"
-import MaterialTable from "material-table"
+import MUIDatatable from "mui-datatables"
 import Typography from "@material-ui/core/Typography"
+import { createMuiTheme, MuiThemeProvider } from "@material-ui/core/styles"
 
 export default function Table({ title, data, columns, marginTop, grouping }) {
+  let getMuiTheme = () =>
+    createMuiTheme({
+      overrides: {
+        MUIDataTable: {
+          responsiveStacked: {
+            width: "100%",
+            overflow: "initial",
+            overflowX: "initial"
+          },
+          paper: {
+            marginBottom: "1rem",
+            marginTop: marginTop != undefined ? "1rem" : null,
+          }
+        },
+        MUIDataTableHeadCell: {
+          fixedHeaderYAxis: {
+            top: 64
+          }
+        }
+      },
+    })
+
   return (
-    <MaterialTable
-      style={{
-        maxWidth: "100%",
-        width: "100%",
-        marginBottom: "1rem",
-        marginTop: marginTop != undefined ? "1rem" : null,
-      }}
-      title={
-        <Typography
-          variant="h6"
-          style={{
-            whiteSpace: "nowrap",
-            overflow: "hidden",
-            textOverflow: "ellipsis",
-            fontFamily: "Saira Semi Condensed",
-          }}
-        >
-          {title}
-        </Typography>
-      }
-      columns={columns}
-      data={data}
-      options={{
-        sorting: true,
-        exportButton: true,
-        pageSize: 25,
-        pageSizeOptions: [25, 50, 100],
-        padding: "dense",
-        emptyRowsWhenPaging: false,
-        exportAllData: true,
-        exportFileName: `${title.replace(" ", "-")}_download`,
-        grouping: grouping === "false" ? false : true,
-      }}
-    />
+    <MuiThemeProvider theme={getMuiTheme()}>
+      <MUIDatatable
+        title={
+          <Typography
+            variant="h6"
+            style={{
+              whiteSpace: "nowrap",
+              overflow: "hidden",
+              textOverflow: "ellipsis",
+              fontFamily: "Saira Semi Condensed",
+            }}
+          >
+            {title}
+          </Typography>
+        }
+        columns={columns}
+        data={data}
+        options={{
+          sort: true,
+          download: true,
+          selectableRows: "none",
+          downloadOptions: {
+            filename: `${title.replace(" ", "-")}_download`,
```

Comment:
This seems to be different than `material-table` as this appears to need the `.csv` extension.

Reply:
Indeed it does... You'd think a button who's tooltip says "download as csv" would automatically add that...